dockerHashing/src/test/scala/cromwell/docker/DockerInfoActorSpec.scala

```diff
@@ -50,6 +52,16 @@ class DockerInfoActorSpec extends DockerRegistrySpec with AnyFlatSpecLike with M
         hash should not be empty
     }
   }
+
+  it should "retrieve a private docker hash on acr" taggedAs IntegrationTest in {
+    dockerActor ! makeRequest("terrabatchdev.azurecr.io/postgres:latest")
+
+    expectMsgPF(15 second) {
+      case DockerInfoSuccessResponse(DockerInformation(DockerHashResult(alg, hash), _), _) =>
+        alg shouldBe "sha256"
+        hash should not be empty
```

Comment:
Huh... doesn't this depend on auth? How is this passing in GHA?

Reply:
The tests tagged `IntegrationTest` don't run on GHA, only locally. Kind of strange, but it definitely comes in handy iterating locally. Presumably in the future we'll have an environment suitable for them, or maybe a more specific `AzureIntegrationTest` tag.

Reply:
TIL!